### PR TITLE
Added missing connectors, shell and innodb cluster. 

### PR DESCRIPTION
**Pull request by miguelaraujo**
Added Connectors: Node.js, C and C++
Added GUI: MySQL Shell
Added HA solution: MySQL InnoDB Cluster

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -133,6 +136,7 @@ This list accepts and encourages pull requests. See [CONTRIBUTING](https://githu
 - [MHA](http://code.google.com/p/mysql-master-ha/) - Master High Availability Manager and tools for MySQL
 - [MySQL Fabric](https://www.mysql.com/products/enterprise/fabric.html) - an extensible framework for managing farms of MySQL Servers.
 - [Percona Replication Manager](https://github.com/percona/percona-pacemaker-agents/) - Asynchronous MySQL replication manager agent for Pacemaker. Supports file and GTID based replication, geo-distributed clusters using booth.
+- [MySQL InnoDB Cluster](https://dev.mysql.com/doc/refman/en/mysql-innodb-cluster-userguide.html) - MySQL native and full stack HA solution.
```
Comment (shlomi-noach) [inline index.md]: This is a reference to a doc page. We're expecting links to repo/code/binaries. The MySQL InnoDB Cluster is just a part of the normal MySQL distribution, hence this link needs to go away.

File: index.md [hunk 2]
```diff
@@ -119,10 +122,10 @@ This list accepts and encourages pull requests. See [CONTRIBUTING](https://githu
...
 - [SQLyog Community edition](https://github.com/webyog/sqlyog-community/wiki/Downloads) - SQLyog Community edition. For Windows, works fine under wine in Mac and Linux
-- [Percona Monitoring and Management](https://www.percona.com/doc/percona-monitoring-and-management/index.html) -  An open-source platform for managing and monitoring MySQL performance. 
+- [Percona Monitoring and Management](https://www.percona.com/doc/percona-monitoring-and-management/index.html) -  An open-source platform for managing and monitoring MySQL performance.
 - [pspg](https://github.com/okbob/pspg) - provides a pager with enhanced visualization and navigation for tabular data. Originally implemented for PostgreSQL, but also supports MySQL.
 - [Ocelot GUI](https://github.com/ocelot-inc/ocelotgui) - GUI client for MySQL or MariaDB, including debugger
-
+- [MySQL Shell](https://dev.mysql.com/downloads/shell/) - Advanced client and code editor for MySQL. Supports development and administration for the MySQL Server with an interactive JavaScript, Python, or SQL interface. Supports the MySQL Document Store (X devAPI) and administration of MySQL InnoDB Cluster (AdminAPI).
```
Comment (shlomi-noach) [inline index.md]: Please make that a one sentence description.

File: index.md [hunk 3]
```diff
@@ -119,10 +122,10 @@ This list accepts and encourages pull requests. See [CONTRIBUTING](https://githu
 - [SequelPro](https://github.com/sequelpro/sequelpro) - a Mac database management application for working with MySQL databases.
 - [mycli](https://github.com/dbcli/mycli) - A Terminal Client for MySQL with AutoCompletion and Syntax Highlighting.
 - [SQLyog Community edition](https://github.com/webyog/sqlyog-community/wiki/Downloads) - SQLyog Community edition. For Windows, works fine under wine in Mac and Linux
-- [Percona Monitoring and Management](https://www.percona.com/doc/percona-monitoring-and-management/index.html) -  An open-source platform for managing and monitoring MySQL performance. 
+- [Percona Monitoring and Management](https://www.percona.com/doc/percona-monitoring-and-management/index.html) -  An open-source platform for managing and monitoring MySQL performance.
```
Comment (shlomi-noach) [inline index.md]: You seem to be adding "Percona Monitoring and Management" twice. Please choose one category only.

**Comment by shlomi-noach** [comment]
Sorry, strike my comment on Percona Monitoring and Management, my confusion.

File: index.md (hunk 1 above)
Comment (miguelaraujo) [reply index.md]: Right, understood. Just to clarify, the link has to be removed but the entry can be kept right?

Comment (miguelaraujo) [reply index.md]: Right, understood. But it is an "High Availability solution" thus I proposed to include it. Link to binaries would imply 3 links: MySQL Server, MySQL Shell and MySQL Router. There are other awesome lists which include links to documentation. Another possibility would be to include instead the following link: https://www.mysql.com/products/enterprise/high_availability.html . Thoughts?

**Comment by miguelaraujo** [comment]
Hi. I addressed the marked issues but also made some extra changes to the pull request. It's also rebased on master.

**Pull request merged**